src/Sources.tsx

```diff
@@ -269,27 +268,6 @@ export const Sources = async (
     if (parseInt(process.env.REACT_APP_SCHEMAVERSION || '100') > 3) {
       if (offline) {
         await offlineSetup();
-      } else {
-        const recs: WorkflowStep[] = (await backup.cache.query(
-          (q: QueryBuilder) => q.findRecords('artifactcategory')
-        )) as any;
-        if (recs.filter((r) => r?.keys?.remoteId).length === 0) {
-          await memory.sync(
-            await remote.pull((q) => q.findRecords('workflowstep'))
-          );
-          await memory.sync(
-            await remote.pull((q) => q.findRecords('artifactcategory'))
-          );
-          await memory.sync(
-            await remote.pull((q) => q.findRecords('artifacttype'))
-          );
-        }
-        const roles: Role[] = (await backup.cache.query((q: QueryBuilder) =>
-          q.findRecords('role')
-        )) as any;
-        if (roles.filter((r) => r?.keys?.remoteId).length < 9) {
-          await memory.sync(await remote.pull((q) => q.findRecords('role')));
-        }
```

Comment:
I am not sure I understand why we are deleting these lines? I think they are needed to pull the values from the DB for these tables, right?

Reply:
checkStaticTables should be loading them so we don't need to do it here